analyzers/tests/SonarAnalyzer.UnitTest/Rules/DoNotShiftByZeroOrIntSizeTest.cs

```diff
@@ -25,18 +25,35 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class DoNotShiftByZeroOrIntSizeTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<DoNotShiftByZeroOrIntSize>();
+
         [TestMethod]
         public void DoNotShiftByZeroOrIntSize() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\DoNotShiftByZeroOrIntSize.cs", new DoNotShiftByZeroOrIntSize());
+            builder.AddPaths("DoNotShiftByZeroOrIntSize.cs").Verify();
 
 #if NET
+
         [TestMethod]
         public void DoNotShiftByZeroOrIntSize_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Console(@"TestCases\DoNotShiftByZeroOrIntSize.CSharp9.cs", new DoNotShiftByZeroOrIntSize());
+            builder.AddPaths("DoNotShiftByZeroOrIntSize.CSharp9.cs")
+                .WithTopLevelStatements()
+                .WithOptions(ParseOptionsHelper.FromCSharp9)
```

Comment:
WithTopLevelStatements already enforces the usage of Csharp9, so no need to add an option specifically for that.